src/clj_sqs_extended/tools.clj

```diff
@@ -1,26 +1,53 @@
 (ns clj-sqs-extended.tools
```

Comment:
these fns look like they all belong to S3 interactions. perhaps call this ns `aws`, or even more specific `s3` instead?

i'm not a fan of generic ns names like tools, helpers, utils, etc.

Reply:
@daisybytes oh I see your PR description. You're on it already. 💯